docs/checks.md

```diff
@@ -588,3 +588,15 @@ possible.
 **Remediation steps**
 - Fix the vulnerabilities. The details of each vulnerability can be found on <https://osv.dev>.
 
+## Webhooks 
+
+Risk: `High`  (possible service can be accessed by third-party)
+
+This check validates if the webhook defined in the repository have a token configured to authenticate the origin of the request to make sure that is a trusted request.
+ 
+
+**Remediation steps**
+- Check if your service supports the token authentication.
```

Comment:
`Check whether your service supports token authentication.` [ADD: how to check this]
`- If there is support for token authentication, set the secret in the webhook configuration. `[can we link to instructions for this?]
`- If the is no support for token authentication, consider implementing it by following [these directions](https://docs.github.com/en/developers/webhooks-and-events/webhooks/securing-your-webhooks).`

Reply:
thanks for the review and feedback, addressed those.
the first one is very dependant on the customer/user hook server.